submissions/Jemikalajah-Philip/Html-Css/Registration-form.html

```diff
@@ -97,9 +97,6 @@ <h2 class = "hero">
         
         </div>
 
-<h3 class="foot">
-        Up4Naija is a new e-commerce platform, with efficient and simple access to buy the best products around  Nigeria. 
-</h3>
-
+<script src="javascript/homepage.js"></script>
```

Comment:
Separation of concerns, why are you importing products js on the registration page?